file_store/src/hex_boost.rs

```diff
@@ -0,0 +1,11 @@
+use chrono::{DateTime, Utc};
+use sqlx::FromRow;
+
+#[derive(Debug, Clone, FromRow)]
```

Comment:
not sure we should be implementing `FromRow` in a file-store struct.  sqlx is now an optional dependency of the file-store project.

Reply:
yeah will address this

Reply:
addressed here 8bd66049471f667490a9cde0d8b029151301d0b0